pkgs/development/tools/backblaze-b2/default.nix

```diff
@@ -39,6 +41,12 @@ python3Packages.buildPythonApplication rec {
     "test_copy_file_by_id"
     "test_integration"
     "test_get_account_info"
+    "test_passing_api_parameters"
```

Comment:
This one I don't think requires network, but it appears to expect some file to be there during setup, and it turns out not to be there. I haven't investigated very deeply, but I was unable to figure out if it's a hidden dependency of the test setup, something we could set up, or pointing to an actual bug in the derivation we build.

Reply:
The issue is that the build environment isn't normally given a "real" `HOME` dir. You can resolve this with:

```
  preCheck = ''
    export HOME=$(mktemp -d)
  '';
```

Reply:
That makes sense, thanks for the tip. That indeed helped with this test, and I was even able to remove a few tests from the list of disabled tests. I just pushed the improved commit.